src/hole.rs

```diff
@@ -75,17 +75,17 @@ impl HoleList {
     /// block is returned.
```

Comment:
Could you update the docs for the new second return type? For example:

"[...] Then the start address of that block and the aligned layout are returned. The automatic layout alignment is required because the `HoleList` has some additional layout requirements for each memory block."